packages/cli/src/web-driver/web-driver.ts

```diff
@@ -1,27 +1,23 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { injectable } from 'inversify';
-import * as Puppeteer from 'puppeteer';
+import Puppeteer from 'puppeteer';
 
 @injectable()
 export class WebDriver {
     public browser: Puppeteer.Browser;
+    public userAgent: string;
 
     constructor(private readonly puppeteer: typeof Puppeteer = Puppeteer) {}
 
     public async launch(chromePath?: string): Promise<Puppeteer.Browser> {
-        let launchOption: Puppeteer.LaunchOptions;
-        launchOption = {
+        this.browser = await this.puppeteer.launch({
             headless: true,
-            timeout: 30000,
+            executablePath: chromePath,
```

Comment:
if chromePath is null, will puppeteer pick up the default or set it to null?

Reply:
It works as it is null by default, we do not pass any value when run app.